### ISSUE TITLE
when i run test using jsdom, I get a ReferenceError: SVGElement is not defined

### ISSUE THREAD

**Issue by forsigner** [developit/preact#444]
``` bash
> jest

 FAIL  __tests__/preact-modal-test.js
  ● preact-modal

    ReferenceError: SVGElement is not defined

      at diff (node_modules/preact/dist/preact.js:155:57)
      at render (node_modules/preact/dist/preact.js:406:16)
      at Object.<anonymous> (__tests__/preact-modal-test.js:15:59)
      at process._tickCallback (internal/process/next_tick.js:103:7)

  ✕ preact-modal (4ms)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        1.34s
```

**Comment by developit** [comment]
Yup! This will be fixed by #423.

**Comment by forsigner** [comment]
Yeah! I know `global.SVGElement = global.Element;` before I opened this Issue,  but it is not Elegant.
Thank for fix it ~

**Comment by developit** [comment]
I find it slightly odd that JSDom doesn't at least provide a passthrough constructor for SVGElement. Preact is built for the browser, and there are no issue with missing SVGElement there. If we can fix it without taking a hit on performance or filesize then that seems fine - just don't want to make people pay for a JSDom workaround ;)

**Comment by developit** [comment]
Fixed in [7.2.0](https://github.com/developit/preact/releases/tag/7.2.0) (beta).

**Comment by marinho** [comment]
I get that same error in version **22.1.4** when trying to use `SVGPathElement`.

This is how my `--debug` did output config:

```
{
  "configs": [
    {
      "automock": false,
      "browser": false,
      "cache": true,
      "cacheDirectory": "/var/folders/lq/ztg_mkn935x1y5gmyw8cfw_80000gn/T/jest_dx",
      "clearMocks": false,
      "coveragePathIgnorePatterns": [
        "/node_modules/",
        "/test/",
        "/dist/",
        "/test/helpers/",
        "/test/test-examples/"
      ],
      "detectLeaks": false,
      "forceCoverageMatch": [],
      "globals": {},
      "haste": {
        "providesModuleNodeModules": []
      },
      "moduleDirectories": [
        "node_modules"
      ],
      "moduleFileExtensions": [
        "ts",
        "tsx",
        "js"
      ],
      "moduleNameMapper": {},
      "modulePathIgnorePatterns": [],
      "name": "e444587e65998f707a150efb3a23fdbc",
      "resetMocks": false,
      "resetModules": false,
      "restoreMocks": false,
      "rootDir": "/Users/setup/Working/my-lib",
      "roots": [
        "/Users/setup/Working/my-lib"
      ],
      "runner": "jest-runner",
      "setupFiles": [],
      "snapshotSerializers": [],
      "testEnvironment": "jest-environment-jsdom",
      "testEnvironmentOptions": {},
      "testLocationInResults": false,
      "testMatch": [],
      "testPathIgnorePatterns": [
        "/node_modules/"
      ],
      "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|tsx|js)$",
      "testRunner": "/Users/setup/Working/my-lib/node_modules/jest/node_modules/jest-jasmine2/build/index.js",
      "testURL": "about:blank",
      "timers": "real",
      "transform": [
        [
          ".(ts|tsx)",
          "/Users/setup/Working/my-lib/node_modules/ts-jest/preprocessor.js"
        ]
      ],
      "transformIgnorePatterns": [
        "/node_modules/"
      ],
      "watchPathIgnorePatterns": []
    }
  ],
  "globalConfig": {
    "bail": false,
    "changedFilesWithAncestor": false,
    "collectCoverage": true,
    "coverageDirectory": "/Users/setup/Working/my-lib/coverage",
    "coverageReporters": [
      "json",
      "text",
      "lcov",
      "clover"
    ],
    "coverageThreshold": {
      "global": {
        "branches": 0,
        "functions": 0,
        "lines": 0,
        "statements": 0
      }
    },
    "detectLeaks": false,
    "expand": false,
    "globalSetup": null,
    "globalTeardown": null,
    "listTests": false,
    "mapCoverage": true,
    "maxWorkers": 3,
    "noStackTrace": false,
    "nonFlagArgs": [],
    "notify": false,
    "passWithNoTests": false,
    "rootDir": "/Users/setup/Working/my-lib",
    "runTestsByPath": false,
    "testFailureExitCode": 1,
    "testPathPattern": "",
    "testResultsProcessor": null,
    "updateSnapshot": "new",
    "useStderr": false,
    "verbose": null,
    "watch": false,
    "watchman": true
  },
  "version": "22.1.4"
}
```

I'm new to Jest, but I based my library on https://github.com/alexjoverm/typescript-library-starter

When compiling with **TypeScript**, everything works fine. It only fails when running tests. Should I paste my `tsconfig.json` too?
